master/contrib/systemd/buildbot@.service

```diff
@@ -1,17 +1,18 @@
 [Unit]
 Description=Buildbot Master
-Wants=network.target
 After=network.target
 
-
 [Service]
 User=buildbot
 Group=buildbot
-WorkingDirectory=/home/buildbot/master
-ExecStart=/usr/local/bin/buildbot start --nodaemon
+WorkingDirectory=/var/lib/buildbot
+ExecStart=/usr/bin/buildbot --verbose start --nodaemon %I
```

Comment:
Can you please document that you will need to create the master (and worker) in a subdirectory named after the systemd instanceID?

Reply:
I've added some commentary to the top of the files that I hope clarifies things. Let me know if there's anything else I can do.